src/webui/components/notification/elementary/RatedComicReaction.tsx

```diff
@@ -0,0 +1,15 @@
+
+import { AggregatedData as ReactionView } from '^/domain/reaction/aggregate/types';
+import { Image, Row, Text } from '^/webui/designsystem';
+
+type Props = {
+    readonly reaction: ReactionView;
```

Comment:
```suggestion
    readonly comicDataUrl: string;
```

Elementary component should get as specific data as possible. This adjustment would also make it consistent with the `RatedPost` component.